platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/SwirldsPlatform.java

```diff
@@ -560,7 +560,7 @@ public class SwirldsPlatform implements Platform, Startable {
         validators.add(new TransactionSizeValidator(transactionConfig.maxTransactionBytesPerEvent()));
         // some events in the PCES might have been created by nodes that are no longer in the current
         // address book but are in the previous one, so we need both for signature validation
-        final List<AddressBook> validationAddressBooks = Stream.of(previousAddressBook, currentAddressBook)
+        final List<AddressBook> validationAddressBooks = Stream.of(currentAddressBook, previousAddressBook)
```

Comment:
I'm not sure about this change.

Reply:
You made the change.  Should it be reverted?

Reply:
I reverted it

Reply:
Why was my change reverted?

Reply:
I did not know you made a change on this branch and just caught the code up with develop.
It's an easy enough fix, but will clear reviews again (and may require another round of trying to get integration tests to run to completion).
Can we do a follow-up PR for this?